deps/build.jl

```diff
@@ -21,4 +21,10 @@ end
     provides(Homebrew.HB, "hdf5", hdf5, os = :Darwin )
 end
 
+begin
+    hdf5 = library_dependency("libhdf5")
```

Comment:
Is the value of `hdf5` important? I think it's only used when passed to `provides`.